armi/nuclearDataIO/cccc/dlayxs.py

```diff
@@ -224,11 +224,43 @@ def __init__(self, fileName, fileMode, dlayxs):
         self.metadata = dlayxs.metadata
 
     def readWrite(self):
-        """Read and write DLAYXS files.
+        r"""Read and write DLAYXS files.
 
         .. impl:: Tool to read and write DLAYXS files.
             :id: I_ARMI_NUCDATA_DLAYXS
             :implements: R_ARMI_NUCDATA_DLAYXS
+
+            Reading and writing DLAYXS delayed neutron data files is performed
+            using the general nuclear data I/O functionalities described in
+            :need:`I_ARMI_NUCDATA`. A DLAYXS file specifically follows the
```

Comment:
Across all the modules, the last sentence reads funny. Consider something like:

"Reading/writing a BLAHBLAH file is performed through the following steps:"